tests/conftest.py

```diff
@@ -145,7 +145,7 @@ async def topic_with_messages(driver, topic_path):
     writer = driver.topic_client.topic_writer(
         topic_path, producer_and_message_group_id="fixture-producer-id"
     )
-    await writer.write_with_ack(
+    await writer.write_with_ack_future(
```

Comment:
The code must use write (and flush on close) or write_with_ack - for wait for commit in the line.

write_with_ack_future will return feature for wait ack, but we don't need it here.